lib/ast-utils.js

```diff
@@ -24,6 +24,8 @@ const bindOrCallOrApplyPattern = /^(?:bind|call|apply)$/;
 const breakableTypePattern = /^(?:(?:Do)?While|For(?:In|Of)?|Switch)Statement$/;
 const thisTagPattern = /^[\s*]*@this/m;
 
+
+const COMMENTS_IGNORE_PATTERN = "^\\s*(?:eslint|jshint\\s+|jslint\\s+|istanbul\\s+|globals?\\s+|exported\\s+|jscs|falls?\\s?through)";
```

Comment:
oh, thanks, I forgot to mention this. I wasn't sure about how to treat that one. @eslint/eslint-team more opinions on this?